static/i18n/en/app.json

```diff
@@ -252,19 +252,19 @@
       "notAvailable": "Service temporarily unavailable, or not available in your country"
     },
     "history": {
-      "disclaimer": "Your swap desktop transactions are not synchronized with the Ledger Live mobile application",
+      "disclaimer": "Your exchange desktop transactions are not synchronized with the Ledger Live mobile application",
```

Comment:
This doesn't seem right @cthepot-ledger can we get a wording check?

Reply:
I agree with you, this doesn't seem right at all. I asked the product team to double-check that all sentences that include the "swap" word are still correct after the "exchange" renaming because of that. The ticket only ask to find and replace "swap" with "exchange" but I think this is not enough, we should take time to double-check all sentences updated and decide if they need to be rewritten or not.